RecoEgamma/EgammaElectronAlgos/BuildFile.xml

```diff
@@ -1,3 +1,4 @@
+<flags   CXXFLAGS="-Ofast"/>
```

Comment:
as I mentioned before, we can expect numerical changes from this flag.
It will be easier if we don't have output changes in the 80X port. 
It's worth keeping it if at least 20% of the observed CPU  gain on the long processing events is from this flag.